chapters/introduction.tex

```diff
@@ -2,32 +2,20 @@ \chapter{Introduction}\label{introduction1}
 
 \section{Overview of Modelica}\label{overview-of-modelica}
 
-Modelica is a language for modeling of physical systems, designed to
-support effective library development and model exchange. It is a modern
-language built on acausal modeling with mathematical equations and
-object-oriented constructs to facilitate reuse of modeling knowledge.
+Modelica is a language for modeling of cyber-physical systems, designed to support effective library development and model exchange.
+It is a modern language built on acausal modeling with mathematical equations and object-oriented constructs to facilitate reuse of modeling knowledge.
```

Comment:
Regarding "modern": This is relative and subject to change. Is Modia more modern cause it was started later in time?

Reply:
Isn't Modelica "built on _mixing_ causal and acausal modeling"?

Reply:
I would not say we really embrace causal modeling with Modelica.  You can do something that _looks_ causal using the `block` style component interfaces, but in the end it's not ascribing causality the way one might think anyway.

Reply:
> Regarding "modern": This is relative and subject to change. Is Modia more modern cause it was started later in time?

Yeah, stating that something is modern in this way is like having _democratic_ in the name of a country.  It looks like an attempt to claim being something that wouldn't be naturally assumed otherwise.

Reply:
@henrikt-ma But next to `block` there are assignment statements `:=` and `algorithm` sections? What about discrete event system modeling (DEVS) is this all done in a non-causal fashion in Modelica?

Reply:
To me, the causal still appears as an exception to the rule in Modelica.  That causality can sometimes be controlled is in my mind not the same as saying that Modelica is a language _for_ causal modeling.

It would be really interesting to hear what others think of this.

Reply:
I agree that dropping 'modern' might be good, and I would suggest that instead of saying that it is "built on acausal modeling with mathematical equations" we could write "supports acausal modeling with mathematical equations", or some variant of this.

To me this provides the main focus on acausal modeling as _differentiator_ compared to other languages.
Saying that a language is "built on" can be less clear as it might also be an internal detail similarly as Scala and Clojure are "built on JVM".

Reply:
I find that a very good and non-controversial suggestion, @HansOlsson.  Done.  Please comment.

Reply:
Looks good to me.